### PR TITLE
feat: automation metadata objects

### PR DESCRIPTION
**Pull request by kandl**
Support automation metadata objects in sdk-model and sdk-backend-spi.
Regenerate api-client-tiger.
Implement new automation service in sdk-backend-tiger.
Add support for dashboard pdf exports in export definitions model.
Align export definitions model with other metadata objects conventions.

risk: low
JIRA: F1-436

---

> [!IMPORTANT]
> Please, **don't forget to run `rush change`** for the commits that introduce **new features** 🙏

---

Refer to [documentation](https://github.com/gooddata/gooddata-ui-sdk/blob/master/dev_docs/continuous_integration.md) to see how to run checks and tests in the pull request. This is the list of the most used commands:

```
extended test - backstop
```

```
extended test - tiger-cypress - integrated
extended test - tiger-cypress - isolated
extended test - tiger-cypress - record
```

### CODE REVIEW

**Review by PavelJiranek — APPROVED** [review]
LGTM

File: libs/sdk-model/src/exportDefinitions/index.ts [hunk 1]
```diff
@@ -46,19 +93,31 @@ export interface IExportDefinitionBase {
...
+ *
+ * @alpha
+ */
+export interface IExportDefinitionMetadataObjectDefinition
+    extends IExportDefinitionBase,
+        IMetadataObjectDefinition {
     type: "exportDefinition";
 }
 
+// also export type IExportDefinitionMetadataObjectDefinition
```
Comment (scavnickyj) [inline libs/sdk-model/src/exportDefinitions/index.ts:111]: Remove

File: libs/sdk-model/src/dashboard/scheduledMail.ts [hunk 2]
```diff
@@ -1,12 +1,14 @@
-// (C) 2020-2022 GoodData Corporation
+// (C) 2020-2024 GoodData Corporation
 import isEmpty from "lodash/isEmpty.js";
 import { IAuditableUsers } from "../base/metadata.js";
 import { ObjRef } from "../objRef/index.js";
 import { IDashboardObjectIdentity } from "./common.js";
 
+// TODO-MKN: remove
```
Comment (scavnickyj) [inline libs/sdk-model/src/dashboard/scheduledMail.ts:7]: Are we keeping this for later?

**Comment by kandl** [comment]
Closing in favor of https://github.com/gooddata/gooddata-ui-sdk/pull/5056

**Pull request closed without merging**